env

```diff
@@ -1,6 +1,7 @@
+FINSCHIA_VERSION="v2.0.1"
```

Comment:
Why you add `FINSCHIA_VERSION` here. It seems not be used.

Reply:
This value should have been used here. This is a bug. I would like to fix this after merging this PR.
https://github.com/Finschia/finschia-wasm/blob/main/.github/workflows/update-and-rebuild.yaml#L22

Reply:
Addressed.
https://github.com/Finschia/finschia-wasm/pull/23/commits/e32351b031c96e4b54150eaf7b576e971d202f8c